src/test/java/api/requests/RequestsAPILoanRenewalTests.java

```diff
@@ -105,32 +104,51 @@ public void allowRenewalLoanByIdWhenFirstRequestInQueueIsHold() throws Interrupt
   }
 
   @Test
-  public void forbidOverrideRenewalLoanByBarcodeWhenFirstRequestInQueueIsRecall() throws InterruptedException, MalformedURLException, TimeoutException, ExecutionException {
+  public void allowRenewalOverrideWhenFirstRequestIsRecall()
+    throws MalformedURLException,
+    InterruptedException,
+    TimeoutException,
+    ExecutionException {
 
     final InventoryItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
     final IndividualResource rebecca = usersFixture.rebecca();
-    final UUID pickupServicePointId = servicePointsFixture.cd1().getId();
 
-    loansFixture.checkOutByBarcode(smallAngryPlanet, rebecca);
+    DateTime loanDueDate = new DateTime(
```

Comment:
Please can this date initialisation be reformatted in a similar manner (and any other in this test class if possible)

Reply:
Sure, I'll do

Reply:
Done